tokens/globals/animation/duration.json

```diff
@@ -3,7 +3,11 @@
     "duration": {
       "small": { "value": 100, "type": "duration", "transform": "duration" },
       "medium": { "value": 250, "type": "duration", "transform": "duration" },
-      "large": { "value": 300, "type": "duration", "transform": "duration" }
+      "medium-expanding": { "value": 250, "type": "duration", "transform": "duration" },
```

Comment:
Probably it makes sense to drop `large` and `medium` and bump a major release for this breaking change